style.scss

```diff
@@ -222,19 +210,286 @@ body>footer {
   @include stripe {
     background-color: #d0d0e8;
   }
-}
-
+}
+
+// =======================================================================================
+// community.html
+
+
+body>#community {
+  >header {
+  min-height: 150px;
+  color: white;
+
+  @include stripe {
+    background-color: #1b4460;
+  }
+
+    >h1 {
+    padding-top: 30px;
+    font-size: 40px;
+    font-weight: 300;
+    text-align: center;
+    }
+
+    >p {
+    text-align: center;
+    color: #c5daff;
+    }
+  }
+}
+
+// =======================================================================================
+// how-it-works.html
+
+body>#how-it-works {

```

Comment:
It looks like there is a lot of similarity between the header styles for each page. We should try to have the common features defined once so that it's easier to make changes.